### ISSUE TITLE
HHVM package fails on debian 7

### ISSUE THREAD

**Issue by pelletiermaxime** [jubianchi/hhvm-cookbook#15]
```
The following packages have unmet dependencies:
 hhvm : Depends: libstdc++6 (>= 4.9.0) but 4.7.2-5 is to be installed
        Depends: gcc-4.9-base but it is not installable
```

I wish I could send you a PR, but haven't found why it requires gcc 4.9 for the package, and no idea how to fix it.

**Comment by pelletiermaxime** [comment]
Just tried again the package install on debian 7 and it now works. They probably fixed it in 3.4.1 or 3.4.2.

**Comment by jubianchi** [comment]
awesome. Thanks a lot!
